src/metrics/graphite_metric.ts

```diff
@@ -28,12 +28,17 @@ export class GraphiteMetric extends AbsractMetric {
   }
 
   getResults(res) {
-    if(res.data[0] === undefined) {
-      throw new Error('data is undefined in response.');
+
+    if(res.data.length < 1) {
```

Comment:
It would be better to check if `res.data !== undefined` first
Otherwise you'd get a `Cannot read property length of undefined` exception